.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - 6
+  - 4
+
+cache:
+  directories:
+    - node_modules
+
+install:
```

Comment:
This line forces the installation to happen with the latest npm version. It overrides the default